packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -516,6 +516,23 @@ abstract class ImageStreamCompleter with Diagnosticable {
     }
   }
 
+  /// Calls all the registered ImageChunkListeners (listeners with an
```

Comment:
```suggestion
  /// Calls all the registered [ImageChunkListeners] (listeners with an
```

Reply:
@goderbauer ImageChunkListener or ImageChunkListeners is not a class or variable or anything on it's own, so there is nothing to link to. Just like 
`If no error listeners (listeners with an [ImageStreamListener.onError] specified) are attached`

I'll rewrite it to image chunk listeners, so it is clear that it is no class.

Reply:
https://master-api.flutter.dev/flutter/painting/ImageChunkListener.html is a typedef that can be linked to.